frontend/src/stories/components/Earthquakes/EarthquakesTable.stories.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import EarthquakesTable from "main/components/Earthquakes/EarthquakesTable";
+import { earthquakesFixtures } from 'fixtures/earthquakesFixtures';
+
+export default {
+    title: 'components/Earthquakes/EarthquakesTable',
+    component: EarthquakesTable
+};
+
+const Template = (args) => {
+    return (
+        <EarthquakesTable {...args} />
+    )
+};
+
+export const Empty = Template.bind({});
```

Comment:
I don't really understand the purpose of binding `Template`'s `this` property to an empty object.